rgf/test/test.py

```diff
@@ -73,6 +71,17 @@ def test_regressor(self):
         print("mse: " + str(mse))
         assert_less(mse, 6.0)
 
+    def test_sample_weight(self):
+        X_train, X_test, y_train, y_test = train_test_split(self.iris.data,
+                                                            self.iris.target,
+                                                            test_size=0.2,
+                                                            random_state=42)
+
+        clf = RGFClassifier()
+        y_pred = clf.fit(X_train, y_train).predict_proba(X_test)
+        y_pred_weighted = clf.fit(X_train, y_train, np.ones(y_train.shape[0])).predict_proba(X_test)
+        self.assertTrue(np.isclose(y_pred, y_pred_weighted).all())
```

Comment:
We also need test with {assert_allclose()`.

I think `nextafter` is usable, because it is numpy supported function.
Another option is fixing `fit` and make it zero input acceptable.
It is beneficial for user.

How do you think?